Codeinwp/neve
# Fix add to cart notice when product is edited with elementor template

### Summary
- The row div that wraps the product content had "nowrap" property because of an issue on the blog with sidebar in safari < 11. 
```
.neve-main > .container > .row {
  flex-wrap: nowrap;
}
``` 
- What I did was to add wrap property on "row" but just on the single product. There is no sidebar on the single product so it shouldn't affect anything. 
```
body.single-product .neve-main > .container > .row {
  flex-wrap: wrap;
}
```

- I also merged the master branch to have the neve's version variable updated.

### Will affect visual aspect of the product
NO


### Test instructions
- Create a single product template with elementor pro
- Click on add to cart and see the notice position


Closes Codeinwp/neve-pro-addon#1163
